packages/binding-opcua/src/opcua_protocol_client.ts

```diff
@@ -0,0 +1,616 @@
+/********************************************************************************
```

Comment:
naming conventions.

you seem to use it inconsistently  "_"  and "- "in the opcua binding package.
We usually use hyphens.
* opcua_protocol_client.ts --> opcua-protocol-client.ts
* client_test.ts --> client-test.ts
* ...

Reply:
Sorry, I am using the opposite convention in node-opcua. I need to play a cameleon and use kebab-case here instead of _snake_case.